src/phoenix/experiments/functions.py

```diff
@@ -105,6 +106,58 @@ def run_experiment(
     dry_run: Union[bool, int] = False,
     print_summary: bool = True,
 ) -> RanExperiment:
+    """
+    Orchestrates running an experiment on a dataset and evaluates the results.
```

Comment:
```suggestion
    Runs an experiment using a given set of dataset of examples.
```
Let's use simpler words